recipe/meta.yaml

```
@@ -97,21 +95,18 @@ outputs:
         - python
         - scipy
```

```suggestion
        - scipy <1.15.0
```

Not currently a problem, but this will avoid future problems just like upstream (xref: https://github.com/single-cell-data/TileDB-SOMA/pull/3445)

---

Thanks @jdblischak ! I'll apply this mod, too, for the 'real' feedstock PR coming up after this pre-check passes (hopefully tonight!)